src/components/Onboarding/Screens/Apps/TokensSettingsBYOT.js

```diff
@@ -0,0 +1,269 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import {
+  Field,
+  GU,
+  Help,
+  Info,
+  IconCheck,
+  IconCross,
+  isAddress,
+  LoadingRing,
+  TextInput,
+  theme,
+} from '@1hive/1hive-ui'
+import Navigation from '../../Navigation'
+import { useOnboardingState } from '@providers/Onboarding'
+import { useTokenData } from '@hooks/useToken'
+import Header from '../../kit/Header'
+
+function useFieldsLayout() {
```

Comment:
What you think of using the same spacing for the fields that we have on the Native screen?

<img width="926" alt="Screen Shot 2021-07-06 at 7 21 41 PM" src="https://user-images.githubusercontent.com/9082013/124673556-66b8aa80-de8f-11eb-831f-99820737cd11.png">

Reply:
vs BYOT
<img width="926" alt="Screen Shot 2021-07-06 at 7 24 45 PM" src="https://user-images.githubusercontent.com/9082013/124673753-d464d680-de8f-11eb-9807-f8bbc4b5a3e5.png">

Reply:
![Screenshot from 2021-07-08 00-06-03](https://user-images.githubusercontent.com/78225502/124834610-59c7b400-df80-11eb-8981-b29c370d5c60.png)

![Screenshot from 2021-07-08 00-04-05](https://user-images.githubusercontent.com/78225502/124834614-5d5b3b00-df80-11eb-8bab-977b09d71503.png)

Made both forms the same width and added a infobox for context.

Maybe should center the loader?

Reply:
Looking great!